ansible/roles/test/tasks/fdb.yml

```diff
@@ -60,5 +60,5 @@
       - router_mac='{{ansible_Ethernet0['macaddress']}}'
       - fdb_info='/root/fdb_info.txt'
       - vlan_ip='{{minigraph_vlan_interfaces[0]['addr']}}'
-    ptf_extra_options: "--relax --debug info --log-file /tmp/fdb_test.FdbTest.{{ansible_date_time.iso8601}}.log "
+    ptf_extra_options: "--platform-dir ptftests --relax --debug info --log-file /tmp/fdb_test.FdbTest.{{ansible_date_time.iso8601}}.log "
```

Comment:
this is not needed since `ptf_platform_dir: ptftests` is already there.

Reply:
Thanks Shuotian,

I removed the extra option.

I also removed the restriction on port 42 since lab team fixed the cabling issue.

Regards,
Ying